src/systems/user_commands/UserCommands.cc

```diff
@@ -725,9 +725,28 @@ bool CreateCommand::Execute()
     }
     case msgs::EntityFactory::kCloneName:
     {
-      // TODO(louise) Implement clone
-      ignerr << "Cloning an entity is not yet supported." << std::endl;
-      return false;
+      auto entityToClone = this->iface->ecm->EntityByComponents(
+          components::Name(createMsg->clone_name()));
```

Comment:
We should check if `kNullEntity != entityToClone` in case the user passes a bad `clone_name`